java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -105,4 +108,61 @@ public interface StreamBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> xdel(String key, String[] ids);
+
+    /**
+     * Returns stream entries matching a given range of IDs.
+     *
+     * @param key The key of the stream.
+     * @param start Starting stream ID bound for range, use {@link IdBound#of} to specify a stream ID,
+     *     or {@link IdBound#ofExclusive} to specify an exclusive bounded stream ID. Use {@link
+     *     InfRangeBound#MIN} to start with the minimum available ID.
+     * @param end Ending stream ID bound for range, use {@link IdBound#of} to specify a stream ID, or
+     *     {@link IdBound#ofExclusive} to specify an exclusive bounded stream ID. Use {@link
+     *     InfRangeBound#MAX>} to end with the maximum available ID.
+     * @return A <code>Map</code> of key to stream entry data, where entry data is an array with pairs
+     *     of item, data.
```

Comment:
```suggestion
     * @return A <code>Map</code> of entry ID to entry data.
```
We don't have pairs there - I mean we don't return data as pairs. I'd like to say that the data is usually flat pairs, but I don't know how.